### ISSUE TITLE
Request: option to disable mouse sensitivity

### ISSUE THREAD

**Issue by nolorfin** [1kc/razer-macos#531]
Is there a way to disable razer-macos's mouse sensitivity settings? 

I use Steermouse to handle my Razer Basilisk Ultimate mouse sensitivity and acceleration. I'm not sure if they overlap my Steermouse settings, and which one takes priority if that's the case.

**Comment by WrathofthePast** [comment]
I don't think it touches the DPI unless you create a profile with a set DPI or you manually set the DPI.
